src/gc/gcpriv.h

```diff
@@ -3002,6 +3002,17 @@ class gc_heap
     PER_HEAP_ISOLATED
     size_t min_segment_size;
 
+    // For SOH we always allocate segments of the same
+    // size unless no gc region requires larger ones.
```

Comment:
Please change 'no gc region' to 'NoGCRegion'. It will be way clearer when parsing this comment later :smile:.